proto/defs.bzl

```diff
@@ -75,15 +53,17 @@ def ts_proto_library(name, proto, **kwargs):
             "--root=%s" % name,
             "--strict-long",  # Force usage of Long type with int64 fields
             "--out=$@",
-            "$(execpaths %s)" % proto_target,
+            "$(locations %s)" % proto_target,
```

Comment:
why locations? perhaps you want `rootpaths`?

Reply:
It fails for external sources when I use `rootpaths` for a strange reason.